src/components/organisms/MediaBox.jsx

```diff
@@ -15,7 +17,13 @@ const Fullscreen = styled.div`
   width: 100%;
 `
 
-export const MediaBox = ({ media }) => {
+const ReactPlayer = styled(RPlayer)`
```

Comment:
Convention we use in Optune is that styled components start with `Styled` .Could be used here too.
`const StyledReactPlayer = styled(ReactPlayer)`